assets/src/models/vehicleStatus.ts

```diff
@@ -119,12 +120,18 @@ export const headwaySpacingToString = (spacing: HeadwaySpacing): string => {
   }
 }
 
-export const statusClass = (status: DrawnStatus): string => {
-  switch (status) {
-    case "plain":
-      return ""
-
-    default:
-      return status
+export const statusClasses = (
+  status: DrawnStatus,
+  settings: UserSettings
```

Comment:
rather than passing in the full `userSettings` I'd rather we pass in just the vehicle adherence color setting here, it'll be self-documentinger that way.

Reply:
FWIW I followed the same pattern used by the `vehicleLabel` helper here, but in general it's probably better to pass the minimum amount of data that the function should depend on.